pkg/commands/git.go

```diff
@@ -270,8 +270,12 @@ func (c *GitCommand) Pull() error {
 }
 
 // Push push to a branch
-func (c *GitCommand) Push(branchName string) error {
-	return c.OSCommand.RunCommand("git push -u origin " + branchName)
+func (c *GitCommand) Push(branchName string, force bool) error {
+	forceFlag := ""
+	if force {
+		forceFlag = "--force-with-lease "
```

Comment:
force with lease means if somebody else has pushed to that branch since you last pushed, your push will be rejected (it's basically a safety measure)